src/support/cli.ts

```diff
@@ -28,14 +28,53 @@ function formatLine (color: Function, type: string, line: string, prefix?: strin
 }
 
 /**
+ * Available log levels.
+ */
+let logLevels: LogLevels = {
+    debug: 0,
+    info: 1,
+    warn: 2,
+    error: 3,
+    silent: 4
+}
+interface LogLevels {
+    debug: number
+    info: number
+    warn: number
+    error: number
+    silent: number
+    [key: string]: number
+}
```

Comment:
Only problem with this is `logLevels.info` no longer seemed to work, are you ok with `loglevels['info']` instead?